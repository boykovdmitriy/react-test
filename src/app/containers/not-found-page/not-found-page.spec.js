import React from 'react';
import NotFoundPage from './index'

describe('notFoundPage', () => {
    it('render notFoundPage component', () => {
        const wrapper = shallow(
            <NotFoundPage/>
        );
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.contains(<section>Page not found</section>)).toEqual(true);
    });
});