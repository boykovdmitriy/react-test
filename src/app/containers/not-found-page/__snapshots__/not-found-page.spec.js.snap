// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notFoundPage render notFoundPage component 1`] = `
<section>
  Page not found
</section>
`;
